treeherder/intermittents_commenter/commenter.py

```diff
@@ -394,7 +394,10 @@ def build_bug_map(self, bugs, option_collection_map):
                 bug["job__option_collection_hash"], "unknown build"
             )
             test_variant = self.get_test_variant(bug["job__signature__job_type_name"])
-            platform_and_build = f"{platform}-{architecture}/{build_type}"
+            if architecture:
+                platform_and_build = f"{platform}-{architecture}/{build_type}"
+            else:
+                platform_and_build = f"{platform}/{build_type}"
```

Comment:
I found we would end up with cases of `test-linux1804-64-qr-/debug` and this was misleading.  it appears that for platform of `linux1804-64-qr`, there is no easy way to get architecture.  Please advise if this is not ideal.

Reply:
I don't see a better solution at the moment. Thank you.